app/views/posts/index.html.erb

```diff
@@ -1 +1,44 @@
-<h1>This is the posts file for testing</h1>
\ No newline at end of file
+<div style="margin: 20px 40px;">
+  <div class="card mb-3" style="border: none;">
+    <div class="row g-0" style="gap: 1.5%;">
+      <div class="col-md-4" style="width: 70px;">
+        <img src ='<%= @user.photo %>' class="img-fluid rounded" alt="User Photo">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body" style="border: 1px solid black; margin-top: 5px; width: 1345px;">
+          <h5 class="card-title"><%= @user.name %></h5>
+          <p class="card-text"><small class="text-muted">
+            Number of posts: <%= @user.posts_counter %></small>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <% @posts.each do |post|%>
+    <div class="card border-success mb-3">
+      <div class="card-body text-success">
+        <a href = <%= user_post_path(post.author.id, post.id) %> style="text-decoration: none;">
+          <h5 class="card-title">Post <%= @posts.index(post) + 1%></h5>
```

Comment:
👏 Great work on your project! One small suggestion for improvement:

- [ ] **Display Actual Post Title:** Currently, you're using placeholder text for each post like "Post 1," "Post 2," etc. For a better user experience, it's recommended to display the actual title of each post. Update your code in the view to show the post title dynamically.
```suggestion
          <h5 class="card-title">Post <%=  post.title %></h5>
```

By implementing this change, you'll provide users with more meaningful information about each post. Keep it up! 🚀👍